style.css

```diff
@@ -0,0 +1,47 @@
+:root {
+    --background-color: white;
+    --text-color: black;
+    --ou-green: rgb(44, 103, 80);
```

Comment:
cutler green is `rgb(0, 104, 72)` in case that's what you're going for https://www.ohio.edu/ucm/ohio-brand/colors

this color is fine but i thought i should put that out there just in case